media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -870,6 +873,8 @@ export default class AssetDetail extends React.Component {
                 defaultControls().forEach(function(control) {
                     me.map.addControl(control);
                 });
+                me.map.addControl(new CenterControl());
```

Comment:
When testing this, the first-time through, everything works. On viewing an image, Zoom & Center are perfect. On click "draw shape", all controls disappear as expected and I'm able to draw a shape on the image. However, the next time through, on "draw shape", the center buttons disappears and the zoom buttons stay. The next time through, the zoom buttons disappear, but the center buttons stay. Let's talk about the best way to tackle this.